### DISCUSSION TITLE
I have a  USBMap for 3020M

### DISCUSSION THREAD

**Discussion by hojokaren** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#59] [General]
I use Google translate

It can run normally on my computer，I don't know if I can help you

You can check if it is correct，@Xtendera

[USBMap_3020m.zip](https://github.com/varszegimarcell/Optiplex-3020-Hackintosh-OpenCore/files/7118754/USBMap_3020m.zip)

![正面](https://user-images.githubusercontent.com/90230793/132281631-8f278af2-90c1-4d0d-84ad-1e8a6fbdd469.jpg)

![背面](https://user-images.githubusercontent.com/90230793/132281651-889b48bc-1f28-4aee-b912-3bf384bc0683.jpg)

**Comment by Xtendera** [comment]
Looking good, @hojokaren! I cannot verify on a real 3020M as I do not have one, but I just made sure there wasn't a virus.command in there. I will upload later as I am a bit busy with homework. You have my thanks.
